app/imports/api/sprints/methods.js

```diff
@@ -24,17 +24,18 @@ export const addSprint = new ValidatedMethod({
     },
 });
 
-export const closeSprint = new ValidatedMethod({
+export const closeSprintToggle = new ValidatedMethod({
     name: 'sprints.close',
```

Comment:
You should change name to `sprints.closeToggle` or `sprints.close` or `sprints.closeOrReopen`.
It's Meteor method name and should be similar to ValidatedMethod name.